src/helpers/helpers.ts

```diff
@@ -1,11 +1,14 @@
 import * as os from 'os';
 import * as path from 'path';
 
+import * as download from 'download';
```

Comment:
I strive to limit micro-dependency bloat in Nativefier. You already have a dep to make easy HTTP calls with a nice API, it's axios, one line below, please remove `download` here and from `package.json` and use axios.